specs/templates/custom-component.md

```diff
@@ -65,13 +65,19 @@
 
 *Work closely with the visual design partner to co-develop the API and anatomy along side the visual design.*
 
+### Native form integration
+
+*Describe the plan for custom element form integration or why it's not necessary.*
+
+*Components that are intended to replace a native form element (input, textarea, select) should generally behave like their native counterpart. See ["More capable form controls" on web.dev](https://web.dev/articles/more-capable-form-controls) for an overview of requirements. Leverage patterns from [FAST Form Associated Custom Elements](https://github.com/microsoft/fast/blob/master/packages/web-components/fast-foundation/src/form-associated/form-associated-custom-element.spec.md).*
+
 ### Angular integration 
 
 *Describe the plan for Angular support, including directives for attribute binding and ControlValueAccessor for form integration. Depending on the contributor's needs, implementing Angular integration may be deferred but the initial spec should still document what work will be needed.*
 
 ### Blazor integration 
 
-*Describe the plan for Blazor support. See the [nimble-blazor CONTRIBUTING.md](/packages/nimble-blazor/CONTRIBUTING.md) for details. Depending on the contributor's needs, implementing Blazor integration may be deferred but the initial spec should still document what work will be needed.*
+*Describe the plan for Blazor support, including form integration. See the [nimble-blazor CONTRIBUTING.md](/packages/nimble-blazor/CONTRIBUTING.md) for details. Depending on the contributor's needs, implementing Blazor integration may be deferred but the initial spec should still document what work will be needed.*
```

Comment:
We don't have great Blazor EditForm support and there is an open issue for that: https://github.com/ni/nimble/issues/766

Maybe remove it from now and add a note to that issue to re-add iit?

Reply:
I started doing that but then noticed that the Blazor CONTRIBUTING does include some basic instructions for form input controls. I updated those instructions to link to the EditForm issue. But I lean towards keeping this addition to remind devs to do those basic steps. Hopefully the link to CONTRIBUTING that's right here will be enough of a breadcrumb.